ext/server/command.cpp

```diff
@@ -222,9 +222,22 @@ void extract_scalar<Tango::DEV_ENCODED>(const CORBA::Any &any, boost::python::ob
         throw_bad_type(Tango::CmdArgTypeName[Tango::DEV_ENCODED]);
     
     bopy::str encoded_format(data[0].encoded_format);
+
+#if PY_MAJOR_VERSION >= 3
+    boost::python::object encoded_data(
+        boost::python::handle<>(
+            PyMemoryView_FromMemory(
```

Comment:
Thanks for working on this.

- As mentioned in the issue thread, we have similar code in `to_py.h`, so I wonder about the difference.
- Second point:  that code doesn't do a Python 3 check, so wonder if we need it here?
      
https://github.com/tango-controls/pytango/blob/19e4c5daf5916384b030b4fb1e373d72d1c0c0e7/ext/to_py.h#L20-L31